src/js/jsx/shared/Dialog.jsx

```diff
@@ -98,7 +98,7 @@ define(function (require, exports, module) {
         },
 
         componentWillMount: function () {
-            this.getFlux().actions.dialog.registerDialog(this.props.id, this._getDismissalPolicy());
+            this.getFlux().store("dialog").registerDialog(this.props.id, this._getDismissalPolicy());
```

Comment:
I feel this is not in line with the recommendations here http://fluxxor.com/documentation/stores.html which suggest that stores should only be updated via the dispatcher.    I do recognize that a) something needs to be done to prevent waiting on the *action* queue for these registrations, and b) that rules are made to be broken.  But there you have it.